week-4/Homework/mandatory/4-groceries.js

```diff
@@ -29,34 +29,48 @@ Exercise 1:
 */
 // Gather all week item names into this array
 let weeklyGroceriesToBuy = [];
+for (let items in weeklyMealPlan) {
+  weeklyGroceriesToBuy.push(weeklyMealPlan[items]);
+}
 
-
+console.log(weeklyGroceriesToBuy);
```

Comment:
this won't work as you want, because it will create an array of arrays
you want to think about a way to join arrays!

Reply:
Used Spread Operator here, to merge all arrays in one.

Reply:
While spread operator is not wrong, but it's too much ES6
easier way maybe
```
weeklyGroceriesToBuy.concat(weeklyMealPlan[days])
```